docs/pages/guides/meshery-metrics.md

```diff
@@ -1,99 +1,195 @@
+</a>
+
+### **Expose `Prometheus` service**
+
+Meshery allows you to expose Prometheus as a service with a single click. You can do this:
+- [Through the Meshery UI](#meshery-ui)
+- Alternatively, you can also attempt [Manual Integration](#manual-steps)
+
+#### **Meshery UI**
+
+Meshery auto-discovers all Prometheus instances available on your local system and will offer you a list of options to choose from. You can select the Prometheus Server that you wish to employ.
 
-a) By default *productpage*  spec type is configured to *ClusterIP*  you can change it to **NodePort** by executing:
+
+<a href="{{ site.baseurl }}/assets/img/meshery-metrics/prometheus-settings.png">
+  <img style="width:500px;" src="{{ site.baseurl }}/assets/img/meshery-metrics/prometheus-settings.png" />
+</a>
+
+Meshery also provides you the option of simply pasting in your Prometheus queries:
+
+- Navigate to the management page for Istio on the Meshery UI
+- Click on <i class="fas fa-caret-right fa-lg"></i>, located under **Apply Custom Configuration** 
+- Paste in your Prometherus query. Click on <i class="fas fa-caret-right fa-lg"></i>
+
+<a href="{{ site.baseurl }}/assets/img/meshery-metrics/prometheus-query.png">
+  <img style="width:500px;" src="{{ site.baseurl }}/assets/img/meshery-metrics/prometheus-query.png" />
+</a>
+
+#### **Manual Steps**
+
+* Get the NodePort of `prometheus` service by executing:
 
 ```
-$ kubectl patch svc grafana -p '{"spec": {"type": "NodePort"}}' -n book-info
+$ kubectl describe services prometheus -n istio-system|grep NodePort
+
+NodePort:  http  30535/TCP
 ```
 
-b) Get NodePort of *productpage* service by executing:
+* Find the `prometheus` endpoint 
+
+The Prometheus endpoint will be *http://$MINIKUBE_IP:NODE_PORT*
+
+```
+http://172.17.0.2:30822
+```
+
+
+#### **Expose Istio BookInfo sample app `productpage` service**
+
+* Get the NodePort of `productpage` service by executing:
 
 ```
 $ kubectl describe services productpage -n book-info|grep NodePort
 
 NodePort:  http  30535/TCP
 ```
 
-c) *productpage* endpoint will be http://$MINIKUBE_IP:NODE_PORT
+* Find the `productpage` endpoint
+
+The `productpage` endpoint will be http://$MINIKUBE_IP:NODE_PORT
 
 ```
 http://172.17.0.2:30535/productpage
 ```
 
-## Linkerd
+#### **Run Performance Tests**
+
+After successfully setting up a connection between your metric service and Meshery, you may proceed to run performance tests by navigating to the Performance Test tab on Meshery:
+
+<a href="/docs/assets/img/performance-management/performance-meshery.png"><img style="width:450px;padding-top:5px;" src="/docs/assets/img/performance-management/performance-meshery.png" /></a>
+
+**Run Test Results**
+
+<a href="/docs/assets/img/performance-management/meshery-and-grafana.png"><img style="width:450px;padding-top:5px;" src="/docs/assets/img/performance-management/meshery-and-grafana.png" /></a>
+
+##### Suggested Reading
```

Comment:
Nice. I'd like to see us considering a "Further Reading" or similar at the end of each doc page. There may be a plugin readily available. Otherwise, a simple system of frontmatter variable will work.

Reply:
@chandrashritii will you create a "help-wanted" issue on this?